content/blog/2019-08-15-new-react-devtools.md

```diff
@@ -49,7 +49,7 @@ Host nodes (e.g. HTML `<div>`, React Native `<View>`) are *hidden by default*, b
 If you are working with React Native version 60 (or older) you can install the previous release of DevTools from NPM:
 
 ```shell
-npm install --dev react-devtools@^3
+npm install --only=dev react-devtools@^3
```

Comment:
How about use `--save-dev`?

Reply:
I think the recommended usage is now `npx react-devtools`. Let's leave this post as is. Thanks!